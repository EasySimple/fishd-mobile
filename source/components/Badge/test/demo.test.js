import webDemoTest from '../../../tests/demoTest';

webDemoTest('Badge');
