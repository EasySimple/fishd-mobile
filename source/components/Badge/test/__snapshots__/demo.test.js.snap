// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Badge demo correctly 1`] = `
<div>
  <div
    class="components-badge-demo-basic"
  >
    <div
      class="demo-title"
    >
      基础用法
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          class="fm-list-item fm-list-item-middle"
        >
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              <span
                class="fm-badge"
              >
                <span
                  style="width:26px;height:26px;background:#ddd;display:inline-block"
                />
                <sup
                  class="fm-badge-dot"
                />
              </span>
              <span
                style="margin-left:12px"
              >
                Dot badge
              </span>
            </div>
            <div
              class="fm-list-extra"
            >
              extra
            </div>
            <div
              aria-hidden="true"
              class="fm-list-arrow fm-list-arrow-horizontal"
            />
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-badge-demo-overflow"
  >
    <div
      class="demo-title"
    >
      展示封顶的数字值
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          class="fm-list-item fm-list-item-middle"
        >
          <div
            class="fm-list-thumb"
          >
            <img
              src="https://zos.alipayobjects.com/rmsportal/faMhXAxhCzLvveJ.png"
            />
          </div>
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              Content
            </div>
            <div
              class="fm-list-extra"
            >
              <span
                class="fm-badge fm-badge-not-a-wrapper"
              >
                <sup
                  class="fm-badge-text"
                >
                  55+
                </sup>
              </span>
            </div>
            <div
              aria-hidden="true"
              class="fm-list-arrow fm-list-arrow-horizontal"
            />
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-badge-demo-corner"
  >
    <div
      class="demo-title"
    >
      置于角落
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          class="fm-list-item fm-list-item-middle"
        >
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              <span
                class="fm-badge fm-badge-corner-wrapper"
              >
                <div
                  class="corner-badge"
                >
                  Use corner prop
                </div>
                <sup
                  class="fm-badge-corner"
                >
                  促
                </sup>
              </span>
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
        <div
          class="fm-list-item special-badge fm-list-item-middle"
        >
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              Custom corner
            </div>
            <div
              class="fm-list-extra"
            >
              <span
                class="fm-badge fm-badge-not-a-wrapper"
              >
                <sup
                  class="fm-badge-text"
                >
                  促
                </sup>
              </span>
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-badge-demo-hidden"
  >
    <div
      class="demo-title"
    >
      text为0时隐藏
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          class="fm-list-item fm-list-item-middle"
        >
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              <span
                class="fm-badge"
              >
                Text 0
              </span>
              <span
                class="fm-badge fm-badge-not-a-wrapper"
              >
                <sup
                  class="fm-badge-text"
                  style="margin-left:12px"
                >
                  new
                </sup>
              </span>
            </div>
            <div
              class="fm-list-extra"
            >
              extra
            </div>
            <div
              aria-hidden="true"
              class="fm-list-arrow fm-list-arrow-horizontal"
            />
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-badge-demo-market"
  >
    <div
      class="demo-title"
    >
      营销样式
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          class="fm-list-item fm-list-item-middle"
        >
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              Marketing:
              <span
                class="fm-badge fm-badge-not-a-wrapper fm-badge-hot"
              >
                <sup
                  class="fm-badge-text"
                  style="margin-left:12px"
                >
                  减
                </sup>
              </span>
              <span
                class="fm-badge fm-badge-not-a-wrapper fm-badge-hot"
              >
                <sup
                  class="fm-badge-text"
                  style="margin-left:12px"
                >
                  惠
                </sup>
              </span>
              <span
                class="fm-badge fm-badge-not-a-wrapper fm-badge-hot"
              >
                <sup
                  class="fm-badge-text"
                  style="margin-left:12px"
                >
                  免
                </sup>
              </span>
              <span
                class="fm-badge fm-badge-not-a-wrapper fm-badge-hot"
              >
                <sup
                  class="fm-badge-text"
                  style="margin-left:12px"
                >
                  HOT
                </sup>
              </span>
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-badge-demo-custom"
  >
    <div
      class="demo-title"
    >
      自定义样式
    </div>
    <div
      class="fm-list"
    >
      <div
        class="fm-list-body"
      >
        <div
          class="fm-list-item fm-list-item-middle"
        >
          <div
            class="fm-list-line"
          >
            <div
              class="fm-list-content"
            >
              Customize
              <span
                class="fm-badge fm-badge-not-a-wrapper"
              >
                <sup
                  class="fm-badge-text"
                  style="margin-left:12px;padding:0 3px;background-color:#f19736;border-radius:2px"
                >
                  券
                </sup>
              </span>
              <span
                class="fm-badge fm-badge-not-a-wrapper"
              >
                <sup
                  class="fm-badge-text"
                  style="margin-left:12px;padding:0 3px;background-color:#21b68a;border-radius:2px"
                >
                  NEW
                </sup>
              </span>
              <span
                class="fm-badge fm-badge-not-a-wrapper"
              >
                <sup
                  class="fm-badge-text"
                  style="margin-left:12px;padding:0 3px;background-color:#fff;border-radius:2px;color:#f19736;border:1px solid #f19736"
                >
                  自动缴费
                </sup>
              </span>
            </div>
          </div>
          <div
            class="fm-list-ripple"
            style="display:none"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
